docs/Explanation/ADR/adr-004-did.md

```diff
@@ -354,6 +354,17 @@ The implementation MUST support the following query parameters:
 
 The format for the queries is:
 -->
+
+## Privacy Considerations
+
+When any data (e.g. W3C Verifiable Credentials) is associated with Cosmos DIDs, sharing that data would also impose sharing the onchain data graph (e.g. transaction history) of the blockchain account that controls the DID.
+
+Using personal identifiable information as DID Method specific identifiers (e.g. alice) discloses personal information every time the DID is shared with a counter party. This specification DOES NOT endorse the use of identifiers that correlates to human beings or other sensible subjects.
```

Comment:
```suggestion
When any data (for example, W3C Verifiable Credentials) is associated with Cosmos DIDs, sharing that data would also impose sharing the on-chain data graph (for example, transaction history) of the blockchain account that controls the DID.

Using personally identifiable information as DID Method-specific identifiers (for example, account name alice) discloses personal information every time the DID is shared with a counterparty. This specification DOES NOT endorse the use of identifiers that correlates to human beings or other sensible subjects.
```

Reply:
let's avoid Latin and spell out "for example" 
I think "alice" is an example of account name alice? or where else would alice be disclosed?